m1well.Expenses/README.md

```diff
@@ -4,8 +4,8 @@ With this plugin you can write down and store your daily/monthly expenses with a
 This is meant to be used for further analysis.
 
 ## Configuration
-For a good start and to get the structure, you can just use the example config,
-which will be added to the NotePlan `_configuration` on the first usage!
+Since Noteplan 3.4 you can use the new plugins settings section.  
+Or you can use the old way with the `_configuration` file.
```

Comment:
The idea is that once people move to v3.4 the migration will be triggered automatically, and then only use the new UI. So downwards compatible for those who have upgraded to v3.4 but not otherwise.